ProcessOptimizer/model_systems/model_system.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+from ProcessOptimizer import expected_minimum
+
+class ModelSystem:
+    """
+    Model System for testing the ProcessOptimizer. Instances of this class 
+    will be used in the example notebooks. 
+
+    Parameters:
+    * `score` [callable]:
+        Function for calculating the score of the system at a given point in 
+        the parameter space. It is expected to have the following signature:
+
+            def score(x, rng=np.random.default_rng(), noise_std=noise_value):
+                # * x is the point in the parameter space where the score will 
+                #   be evaluated. 
+                # * rng is the random number generator used for adding noise 
+                #   to the system. The user can set a seed through this 
+                #   parameter. There must be a default value for this parameter. 
+                # * noise_std is the standard deviation of the noise. There
+                #   must be a default value for this parameter. 
+                ....
+                # The score of the system is returned
+                return score
+
+    * `space` [Space]:
+        The parameter space in the form of a Space object. 
+
+    * `true_min` [array with dimensions matching the parameter space]:
+        The true minimum value of the score function within the parameter space. 
```

Comment:
Since you're stating that this needs to be an array of size N = space.n_dims(), I am guessing that the user needs to supply the _location_ of the true minimum, not the value? If you mean the actual best result obtainable in the system, the input should simply be a float?

Reply:
This is an error in the comment. It is the value that should be supplied, not the location. Therefore it should be a float. I will correct the comment.